validation/vertical_mixing_closures/bottom_boundary_layer.jl

```diff
@@ -0,0 +1,66 @@
+using GLMakie
+using Oceananigans
+using Oceananigans.Units
+using Printf
+
+using Oceananigans.TurbulenceClosures.CATKEVerticalDiffusivities:
+    CATKEVerticalDiffusivity, MixingLength
+
+# Parameters
+Δz = 4          # Vertical resolution
+Lz = 256        # Extent of vertical domain
+Nz = Int(Lz/Δz) # Vertical resolution
+f₀ = 0.0        # Coriolis parameter (s⁻¹)
```

Comment:
Why so verbose to explicitly denote that coriolis parameter is zero?

Reply:
To show users how it might be changed

Reply:
Good point!